Cargo.toml

```diff
@@ -95,11 +96,15 @@ tls-none = []
 runtime-async-std-native-tls = ["runtime-async-std", "tls-native-tls"]
 runtime-async-std-rustls = ["runtime-async-std", "tls-rustls-ring"]
 
+runtime-smol-native-tls = ["runtime-smol", "tls-native-tls"]
+runtime-smol-rustls = ["runtime-smol", "tls-rustls-ring"]
```

Comment:
The existing combination features have been soft-deprecated for a long time now. I'd rather remove them entirely at this point than add new ones.